docs(fern): add SEO meta descriptions + headlines to nightly docs (#1589) (#1600)
NVIDIA-NeMo/Gym: fern/versions/latest/pages/agent-server/index.mdx

---
title: "Agent Server"

> description: "The NeMo Gym Agent server orchestrates rollouts: it runs the agent loop, calls the Model server, routes tool calls to Resources, and collects the final reward."

Reviewer: ```suggestion
description: "The NeMo Gym Agent server houses the agent harness, which wraps a model with an agent loop."
```
